src/Microsoft.DotNet.Darc/src/Darc/Operations/GetSubscriptionsOperation.cs

```diff
@@ -52,40 +51,60 @@ public override async Task<int> ExecuteAsync()
                     Console.WriteLine($"{subscription.SourceRepository} ({subscription.Channel.Name}) ==> '{subscription.TargetRepository}' ('{subscription.TargetBranch}')");
                     Console.WriteLine($"  - Id: {subscription.Id}");
                     Console.WriteLine($"  - Update Frequency: {subscription.Policy.UpdateFrequency}");
```

Comment:
>Policy [](start = 76, length = 6)

Do we need to check if `Policy` is null before using it or it'll always be set?

Reply:
Policy is always non-null